### ISSUE TITLE
The Rope sometimes gets caught mid-air, into very small and sometimes invisible terrain fragments

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#43]
```
What steps will reproduce the problem?
1. Using the Rope
2. After collecting a crate the rope is still inflicted by it

What is the expected output? What do you see instead?
Only jam when the terrain fragment is bigger (and/or visible)

What version of the product are you using? On what operating system?
0.9.13, Win7


```

-----
Original issue reported on code.google.com by `joship...@gmail.com` on 15 Sep 2010 at 4:11

**Comment by GoogleCodeExporter** [comment]
```
This shouldn't happen to invisible parts of the terrain (maybe just almost 
invisible due to the theme used?). But it's right that the rope can get stuck 
inside terrain no longer releasing the "node".
```

-----
Original comment by `mario.li...@gmail.com` on 16 Sep 2010 at 9:05
* Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
yes, exactly, and that can be annoying, especially when you would have needed 
the whole rope-length to do a jump or so
```

-----
Original comment by `joship...@gmail.com` on 16 Sep 2010 at 9:30

**Comment by GoogleCodeExporter** [comment]
```
The problem described (rope affected by a collected crate) is a bit different 
from this thing of "invisible fragments" - anyway, there's a forum thread on 
it, some argument on (how/whether) it should be fixed and a patch that someone 
submitted that unc0rr said didn't properly address the situation.

```

-----
Original comment by `kyberneticist@gmail.com` on 21 Sep 2010 at 9:55

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 4378:ab577db125c4 and (probably) 3545:066faceb977d

```

-----
Original comment by `kyberneticist@gmail.com` on 21 Nov 2010 at 6:23
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Verified (rope works everywhere).
```

-----
Original comment by `jerryma1...@gmail.com` on 26 Nov 2011 at 7:59
* Changed state: **Verified**
